### PR TITLE
Add a new exercise

### PR DESCRIPTION
**Pull request by MehmetTerc**

### CODE REVIEW

File: src/DB/exerciseData.json [hunk 1]
```diff
@@ -312,5 +312,16 @@
   "videoLink": " https://www.youtube.com/watch?v=fIWP-FRFNU0 ",
   "gh-name": "B2prakash",
   "image": "https://media.tenor.com/m2Dfyh507FQAAAAC/8preacher-curl.gif"
+},
+{
+  "exercise": "Romanian Deadlift",
+  "instructions": [
+    "Stand with feet shoulder-width apart, hinge at the hips while keeping a straight back,"
```
Comment (gabrysia694) [inline src/DB/exerciseData.json]: You're missing a comma here

File: src/DB/exerciseData.json [hunk 2]
```diff
@@ -312,5 +312,16 @@
...
+    "lower the weight close to the body, and drive through the heels to return. Maintain a "
```
Comment (gabrysia694) [inline src/DB/exerciseData.json]: And one more comma here

**Review by gabrysia694 — CHANGES_REQUESTED** [review]
Please remove the last sentence of your instruction as it describes the effect of the exercise rather than providing a step on how to correctly perform the exercise.

**Comment by MehmetTerc** [comment]
@gabrysia694 I correct my mistakes, can you merged it

File: src/DB/exerciseData.json [hunk 3]
```diff
@@ -316,9 +316,8 @@
 {
   "exercise": "Romanian Deadlift",
   "instructions": [
-    "Stand with feet shoulder-width apart, hinge at the hips while keeping a straight back,"
-    "lower the weight close to the body, and drive through the heels to return. Maintain a "
-    "neutral spine. Strengthen hamstrings, glutes, and lower back with this effective exercise."
+    "Stand with feet shoulder-width apart, hinge at the hips, while keeping a straight back,"
```
Comment (gabrysia694) [inline src/DB/exerciseData.json]: You're missing a comma right here, should be `"Stand with feet shoulder-width apart, hinge at the hips, while keeping a straight back,",`

**Review by gabrysia694 — CHANGES_REQUESTED** [review]

**Review by gabrysia694 — APPROVED** [review]

**Comment by MehmetTerc** [comment]
i correct the missing comma

**Pull request merged**